frontend/model/contracts/group.js

```diff
@@ -539,8 +586,7 @@ DefineContract({
         reason: string
       }),
       process ({ data, meta, contractID }, { state, getters }) {
-        state.profiles[meta.username].status = PROFILE_STATUS.REMOVED
-        state.profiles[meta.username].departedDate = meta.createdDate
+        memberLeaves(state, meta.username, meta.createdDate)
```

Comment:
Again, I think maybe the following some lines got accidentally deleted when adding this line:
```
        state.profiles[meta.username].status = PROFILE_STATUS.REMOVED
        state.profiles[meta.username].departedDate = meta.createdDate
```
@taoeffect Can you confirm that these lines were not removed by mistake?